### ISSUE TITLE
Asking for type info when code fails to compile causes ghc-modi crash

### ISSUE THREAD

**Issue by joprice** [carymrobbins/intellij-haskforce#261]
If there is a compilation error, then type info is requested, ghci crashes. Restarting ghc-modi does not seem to have any effect after that.

```
Killing ghc-modi due to process failure.<br/><br/>
You can restart it using <b>Tools &gt; Restart ghc-modi</b><br/><br/>
Executing ghc-modi command 'type /Users/josephprice/dev/github.com/joprice/groom/app/Main.hs 24 19' 
failed with error: EXCEPTION: types:
.... 
```

**Comment by carymrobbins** [comment]
Hey @joprice!  I've ran into this as well, and it's annoying.  I'm thinking this should be a pretty straightforward fix.

Also, for me restarting ghc-modi fixes the problem until I attempt to show type info again if the module has a compile error.

**Comment by carymrobbins** [comment]
This should be fixed in the upcoming release, **v0.3-beta.31**.  I should have it out this evening.
